### PR TITLE
Try enabling Python 3.10 support in github actions / tox

### PR DESCRIPTION
**Pull request by ctheune**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py35, py36, py37, py38, py39, python3.10, pre-commit
+    py35, py36, py37, py38, py39, py3.10, pre-commit
```
Comment (jugmac00) [inline tox.ini:3]: it should be just `py310`

Comment (ctheune) [reply tox.ini:3]: Gnah. I stared at the relevant tox change and got it completely wrong ... le sigh ... Thanks. 

It's broken anyways, so ...

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
   build:
     strategy:
       matrix:
-        python-version: [ '3.6', '3.7', '3.8', '3.9' ]
+        python-version: [ '3.6', '3.7', '3.8', '3.9', '3.10.0-beta.1' ]
```
Comment (jugmac00) [inline .github/workflows/main.yml:19]:
```suggestion
        python-version: [ '3.6', '3.7', '3.8', '3.9', '3.10' ]
```

**Comment by gforcada** [comment]
Hi! 👋🏾 is there any way I can help on this 3.10 porting effort? 😄

**Comment by jugmac00** [comment]
FWIW I (sadly) have no more chance to use baton, so all efforts should be communicated with @ctheune

**Comment by gforcada** [comment]
I rebased the branch on top of current `main` in my [local fork of this repo](https://github.com/gforcada/batou) and pushed [the branch](https://github.com/gforcada/batou/tree/161-support-python-3.10).

I got conflicts with the lock files on the `examples` folder, how is one meant to update them? is it important to do it for this PR to be accepted? 🤔 

Should I open a new PR with my branch? I have no rights in this repo, so I can not push the changes in the branch used for this PR 😓

**Comment by elikoga** [comment]
https://github.com/flyingcircusio/batou/pull/452 closes this

**Pull request closed without merging**